platform/src/Playground.js

```diff
@@ -762,7 +774,13 @@ function handleResponseActionFunction(action, requestPromise){
                 switch (action.outputType){
                     case "code":
                         // Text
+                        var editor = outputPanel.getEditor();
```

Comment:
Why do we need to keep this logging code?

Reply:
Agree this should be removed.

Reply:
Not needed. It was used for debugging purposes.

Reply:
Removed debug logs in 92d27f5df4c96eba4476c6af1fb71bf9ad925572